README.md

```diff
@@ -1,2 +1,50 @@
 
-NoSql Assignment
+NoSQL Assignment 
+Users 
+ 
+{ 
+_id: “username”, 
```

Comment:
- username should be a separate field, _id should simply hold the ObjectId that is provided by mongo 
- Object ID is already indexed by mongo.